### PR TITLE
[OSDEV-974] Reporting. Contributor type by %.

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-974](https://opensupplyhub.atlassian.net/browse/OSDEV-974)  Reporting. Contributor type by %.

* Admin see in the report data for the percent of data contributors on the platform by type (this should be in percent format with two decimal places shown), only accounts that have contributed data, the data should be ordered by most recent to oldest month and display mid-month values.

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-660](https://opensupplyhub.atlassian.net/browse/OSDEV-660) - Remove punctuation issues with duplicated commas and double quotes while facility list uploading.
 
 ### What's new
+* [OSDEV-974](https://opensupplyhub.atlassian.net/browse/OSDEV-974) Reporting. Contributor type by %. Admin see in the report data for the percent of data contributors on the platform by type (this should be in percent format with two decimal places shown), only accounts that have contributed data, the data should be ordered by most recent to oldest month and display mid-month values.
```
Comment (VadimKovalenkoSNF) [inline doc/release/RELEASE-NOTES.md:33]: Fix typo, it should be `Admin sees`

File: src/django/api/reports/contributor_type_by_percent.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
+SELECT
```
Comment (VadimKovalenkoSNF) [inline src/django/api/reports/contributor_type_by_percent.sql:1]: I think this query can be optimized, but need other developer to take a look.

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]
Left few comments

File: src/django/api/reports/contributor_type_by_percent.sql (hunk 2 above)
Comment (VadimKovalenkoSNF) [reply src/django/api/reports/contributor_type_by_percent.sql:12]: Looks better now, thanks

**Review by roman-stolar — APPROVED** [review]
Approved

**Pull request merged**